validator/riscv/rv32_validator.cc

```diff
@@ -123,6 +124,13 @@ rv32_validator_t::rv32_validator_t(meta_set_cache_t *ms_cache,
   csr_tags.reset(m_to_t(ms));
   ms = ms_factory->get_meta_set("ISA.RISCV.Reg.Env");
   pc_tag = m_to_t(ms);
+  // set initial tags for specific CSRs
+  ms = ms_factory->get_meta_set("ISA.RISCV.CSR.MEPC");
+  csr_tags[CSR_MEPC] = m_to_t(ms);
+  ms = ms_factory->get_meta_set("ISA.RISCV.CSR.MTVAL");
```

Comment:
As @ccasin mentioned, we want to keep the existing naming convention for CSRs. Can you change `MTVAL` and `MTVEC` to `MTVal` and `MTVec`?

Reply:
I'm happy to change this back--what should I change mepc to?